src/qt/test/wallettests.cpp

```diff
@@ -391,6 +418,140 @@ void TestGUI(interfaces::Node& node, const std::shared_ptr<CWallet>& wallet)
+    QVERIFY(startButton->isEnabled());
+
+    // Test starting the deniabilization process
+    CAmount balance = walletModel.wallet().getBalance();
+    const uint expectedTxSize = 304; // 2 utxo inputs, 2 txout (LEGACY output type)
+    CAmount expectedTxFee = walletModel.wallet().getRequiredFee(expectedTxSize);
+
+    // We expect to get a confirmation message box and we want to OK it
+    bool clickedOk = false;
+    ConfirmMessageBox(&clickedOk, QMessageBox::Ok);
+    startButton->click();
+    qApp->processEvents();
+    // Verify we did get the confirmation message box
+    QVERIFY(clickedOk);
+
+    // Verify only the stop button is enabled
+    QVERIFY(stopButton->isEnabled());
+    QVERIFY(!startButton->isEnabled());
+    QVERIFY(!hourlyRadioButton->isEnabled());
+    QVERIFY(!dailyRadioButton->isEnabled());
+    QVERIFY(!weeklyRadioButton->isEnabled());
+    // FIXME: AmountSpinBox doesn't correctly intercept isEnabled()()
+    // QVERIFY(!budgetSpinner->isEnabled());
+
+    // Verify a deniabilization transaction was processed and the before and after balances match (minus the tx fee)
+    CAmount balanceAfterTx = walletModel.wallet().getBalance();
+    QCOMPARE(balance, balanceAfterTx + expectedTxFee);
+
+    // Verify the budget was updated correctly
+    QCOMPARE(budgetSpinner->value(), budget - expectedTxFee);
+
+    // Verify status shows we're waiting for the tx to be confirmed
+    std::string statusTextAfterTx = statusLabel->text().toStdString();
+    QCOMPARE(statusTextAfterTx, std::string("Waiting for the deniabilization transaction to be confirmed..."));
+
+    // Stop the deniabilization processing
+    stopButton->click();
+    qApp->processEvents();
+
+    // Verify stop is disabled and everything else is re-enabled
+    QVERIFY(!stopButton->isEnabled());
+    QVERIFY(startButton->isEnabled());
+    QVERIFY(hourlyRadioButton->isEnabled());
+    QVERIFY(dailyRadioButton->isEnabled());
+    QVERIFY(weeklyRadioButton->isEnabled());
+    // FIXME: AmountSpinBox doesn't correctly intercept isEnabled()()
+    // QVERIFY(budgetSpinner->isEnabled());
+
+    // Bump the min fee to test fee bumping
+    wallet->m_min_fee = CFeeRate(10000);
+    CAmount expectedBumpedTxFee = walletModel.wallet().getRequiredFee(expectedTxSize);
+
+    // Restart the deniabilizaiton process
+    startButton->click();
+    qApp->processEvents();
+
+    CAmount balanceAfterFeeBumpTx = walletModel.wallet().getBalance();
+    QCOMPARE(balance, balanceAfterFeeBumpTx + expectedBumpedTxFee);
+
+    // Verify the budget was updated correctly
+    QCOMPARE(budgetSpinner->value(), budget - expectedBumpedTxFee);
+
+    // mine a block to confirm the transaction
+    MineBlock(test.m_node, GetScriptForRawPubKey(test.coinbaseKey.GetPubKey()));
+    {
+        LOCK(wallet->cs_wallet);
+        wallet->SetLastBlockProcessed(106, WITH_LOCK(node.context()->chainman->GetMutex(), return node.context()->chainman->ActiveChain().Tip()->GetBlockHash()));
```

Comment:
Crosses even column 160.
```suggestion
        wallet->SetLastBlockProcessed(106, WITH_LOCK(
                node.context()->chainman->GetMutex(),
                return node.context()->chainman->ActiveChain().Tip()->GetBlockHash()));
```
Wouldn't reach even column 100.

Reply:
There was a bug introduced in the fee bumping after the last rebase. Interestingly it didn't get flagged by the compiler ... even though it was an obviously wrong type being assigned to CRecipient struct ... :-O 
At any rate, I fixed it in the latest commit. Please let me know if you're still seeing an issue running the tests.
Thanks!